src/components/Releases/Releases.vue

```diff
@@ -79,6 +79,7 @@ import ReleaseCard from './ReleaseCard.vue';
 import SprintCard from '../Sprints/SprintCard.vue';
 import AddSprint from '../Sprints/AddSprint.vue';
 import dateConvert from '../../mixins/dateConvert';
+import { release } from 'os';
```

Comment:
Where this import come from?

Reply:
This import is not necessary, I will remove it